client/signup/steps/plans/index.jsx

```diff
@@ -46,9 +47,10 @@ class PlansStep extends Component {
 			processingMessage: isEmpty( cartItem )
 				? translate( 'Free plan selected' )
 				: translate( 'Adding your plan' ),
-			stepName: stepName,
-			stepSectionName: stepSectionName,
-			cartItem
+			stepName,
+			stepSectionName,
+			cartItem,
+			...additionalStepData
```

Comment:
I'm unsure why to need to add both `additionalStepData` and `... additionalStepData`.